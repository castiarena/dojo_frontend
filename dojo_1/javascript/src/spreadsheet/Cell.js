export default class Cell {
    constructor(){

    }

    render() {
        return '<td></td>';
    }
}