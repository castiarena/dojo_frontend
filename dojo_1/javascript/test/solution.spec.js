import SpreadSheet from '../src/spreadsheet';
import Cell from '../src/cell';


describe('Bateria de tests para Spreadsheet', () => {

    it('Componente spreadsheet renderiza una tabla vacÃ­a', () => {
        const spreadSheet = new SpreadSheet();
        expect(spreadSheet.render()).toMatchSnapshot();
    });

    it('Renderizar una tabla con header', () => {
        const spreadSheet = new SpreadSheet(
            [
                new Cell(), new Cell()
            ], [

            ]);

        expect(spreadSheet.render()).toMatchSnapshot();

    });

    it('Renderizar una tabla con header de cuatro columnas', () => {

    });


});