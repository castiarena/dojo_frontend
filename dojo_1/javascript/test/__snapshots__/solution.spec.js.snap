// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bateria de tests para Spreadsheet Componente spreadsheet renderiza una tabla vac√≠a 1`] = `"<table></table>"`;

exports[`Bateria de tests para Spreadsheet Renderizar una tabla con header 1`] = `"<table><thead><tr><td></td><td></td></tr><thead></table>"`;
